src/components/30-organisms/commercial-hero-banner/index.react.d.ts

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+type Variant = 'light' | 'dark';
+type ImagePosition =
+  | 'center center'
+  | 'top left'
+  | 'top right'
+  | 'bottom right'
+  | 'bottom left';
+
+interface AXACommercialHeroBannerProps {
+  className?: string;
+  variant?: Variant;
+  imagePosition: ImagePosition;
```

Comment:
why is there no `imagePosition` prop in `index.js` but for react you can set an `imagePosition` prop?

Reply:
Shit, I forgot to remove it, thanks!